index.js

```diff
@@ -220,16 +220,23 @@ class Analytics {
       callback(err, data)
     }
 
+    // Don't set the user agent if we're not on a browser. The latest spec allows
+    // the User-Agent header (see https://fetch.spec.whatwg.org/#terminology-headers
+    // and https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/setRequestHeader),
+    // but browsers such as Chrome and Safari have not caught up.
+    const headers = {}
+    if (typeof window === 'undefined') {
+      headers['user-agent'] = `analytics-node/${version}`
+    }
```

Comment:
The comment and PR title don't match the implementation.  This implementation seems to set the `user-agent` header only if we're **not** on a browser (`window === undefined` means we're not in a browser).